app/views/user/show/gamesContent.scala

```diff
@@ -36,15 +36,15 @@ object gamesContent {
           if (pager.nbResults > 0)
             frag(
               div(cls := "search__status")(
-                strong(pager.nbResults.localize, " games found")
+                strong(trans.gamesFound(pager.nbResults.localize))
```

Comment:
should be `trans.gamesFound.plural(pager.nbResults, pager.nbResults.localize)`

Reply:
done